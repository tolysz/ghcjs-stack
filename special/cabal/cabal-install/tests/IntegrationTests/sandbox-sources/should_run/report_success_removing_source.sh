. ../common.sh

# Create the sandbox
cabal sandbox init > /dev/null

# Add the sources
cabal sandbox add-source p > /dev/null
cabal sandbox add-source q > /dev/null

# Remove one of the sources
cabal sandbox delete-source p q
