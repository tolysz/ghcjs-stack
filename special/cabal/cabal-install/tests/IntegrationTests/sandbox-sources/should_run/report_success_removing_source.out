Success deleting sources: "p" "q"

Note: 'sandbox delete-source' only unregisters the source dependency, but does
not remove the package from the sandbox package DB.

Use 'sandbox hc-pkg -- unregister' to do that.
